### PR TITLE
Added Template for Aggregated Employer Demand Web

### PR DESCRIPTION
**Pull request by Knyatsine**

### CODE REVIEW

File: azure/template.json [hunk 1]
```diff
@@ -24,18 +24,40 @@
...
+        "sharedEnvResourceGroup": {
+            "type": "string"
         }
     },
     "variables": {
         "deploymentUrlBase": "https://raw.githubusercontent.com/SkillsFundingAgency/das-platform-building-blocks/master/templates/",
         "resourceNamePrefix": "[toLower(concat('das-', parameters('resourceEnvironmentName'),'-', parameters('serviceName')))]",
-        "resourceGroupName": "[concat(variables('resourceNamePrefix'), '-rg')]"
+        "resourceGroupName": "[concat(variables('resourceNamePrefix'), '-rg')]",
+        "appServiceName": "[concat(variables('resourceNamePrefix'),'as')]"
```
Comment (NickGraham101) [inline azure/template.json:60]: the string `as` needs to start with a dash, ie `-as`

File: azure/template.json [hunk 2]
```diff
@@ -46,12 +68,143 @@
             "tags": "[parameters('tags')]",
             "properties": {
             }
+        },
+        {
+            "type": "Microsoft.Resources/deployments",
+            "apiVersion": "2020-06-01",
+            "name": "app-service-certificate",
+            "resourceGroup": "[variables('resourceGroupName')]",
```
Comment (NickGraham101) [inline azure/template.json:76]: the certificate should be created in the shared environment resource group, the same group the app service plan is in

File: azure/template.json [hunk 3]
```diff
@@ -46,12 +68,143 @@
...
+                                {
+                                    "name": "APPINSIGHTS_INSTRUMENTATIONKEY",
+                                    "value": "[reference('application-insights').outputs.InstrumentationKey.value]"
+                                },
+                                {
+                                    "name": "ASPNETCORE_ENVIRONMENT",
+                                    "value": "[toUpper(parameters('environmentName'))]"
+                                },
+                                {
+                                    "name": "WEBSITE_LOAD_CERTIFICATES",
```
Comment (NickGraham101) [inline azure/template.json:160]: This app setting isn't usually added.  It allows the application code access to the certificate store but that's not commonly a requirement for our apps.  Has this been specifically requested?

File: azure/template.json [hunk 4]
```diff
@@ -46,12 +68,143 @@
...
     "outputs": {
         "ResourceGroupName": {
             "type": "string",
             "value": "[variables('resourceGroupName')]"
+        },
+        "AppServiceName": {
+            "type": "string",
+            "value": "[variables('appServiceName')]"
+        },
+        "CertificateThumbprint": {
```
Comment (NickGraham101) [inline azure/template.json:205]: Is the certificate thumbprint needed by the release pipeline?  If not there's no need to output it.

File: azure/template.json (hunk 3 above)
Comment (Knyatsine) [reply azure/template.json:160]: @NickGraham101 This hasn't been requested. I will remove it

**Review by NickGraham101 — APPROVED** [review]

**Pull request merged**